app/public/app.webmanifest

```diff
@@ -0,0 +1,23 @@
+{
+    "lang": "en",
+    "dir": "ltr",
+
+    "name": "Kards",
+    "short_name": "Kards",
+    "id": "kards",
+    "description": "Socially social: Kards.",
+
+    "icons": [{
+        "src": "LogoSmall.svg",
```

Comment:
Do android and ios both "like" svg's? png might be a saver option but could be wrong.

Reply:
PNGs would require multiple files with separate sizes, unless there is some resizing algorithm in Next than can warrant grabbing resized images automatically. If those options are available, would require some more implementation.

Reply:
For the sake of safety, added PNG icons.